examples/slash_commands/main.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"flag"
+	dgo "github.com/bwmarrin/discordgo"
```

Comment:
Is import aliasing a good idea? I don't see any reason to do so as there's no conflicts at all.
Formatting it similarly to existing examples would be a better idea.